plugins/populator/frontend/public/localization/populator.properties

```diff
@@ -19,18 +19,21 @@ populator.requests = Requests generated
 populator.push-users = Messaging enabled users
 populator.purchases = Purchases made
 populator.crashes = Crashes occured
-populator.maxtime = Max time to run (sec)
-populator.warning1 = This plugin populates data for current app
-populator.warning2 = Make sure your app doesn't involve real data,<b>otherwise it will conflict with generated data</b>
+populator.maxtime = Maximum time to run (seconds)
+populator.warning1 = This plugin populates data for {0}
+populator.warning2 = Make sure your app doesn't involve real data,otherwise it will conflict with generated data.
 populator.help = <p>Select the start and end dates of user activities and press "Start Generating"</p><p>Countly populator will generate user data and emulate their behaviour with sessions, events, etc</p><p>The longer you let populator run, the more frequent sessions and events users will have in the selected time period</p><p>To stop process, simply click "Stop Generating" button</p>
 populator.yes-populate-data = Yes, populate data
+populator.no-populate-data = No, don't do that
 populator.populate-data-for = Populate data for {0}?
 populator.demo-data = Demo data
 populator.tooltip = Populate app with random data for demo after creation
 populator.success = Finished populating data. Click on Continue to refresh the page and go to dashboard.
-populator.generating = Generating data... 
+populator.generating = Generating ... 
 populator.processing = Still processing some data...
 populator.jobs = Wating for background jobs to complete... <br/> (can take up to several minutes)
 populator.done = Done!
```

Comment:
Can you please cross reference localization strings in the populator that are not used anymore and remove them from main localization file?